src/helpers/makeExecuteLatest/index.js

```diff
@@ -2,6 +2,7 @@ export default () => {
   const ids = new Set()
 
   return {
+    addSync: callback => (...args) => callback(...args),
```

Comment:
Also here it could be:
```addSync: callback => args => callback(args),```

Reply:
actually i put it there by mistake, sorry :) i didn't used it, removing :)